internal/common/common.go

```diff
@@ -2,6 +2,17 @@ package common
 
 import "time"
 
+const (
```

Comment:
@bzwei Wouldn't this have to change
https://github.com/RedHatInsights/rhc-worker-catalog/blob/367eee14b8dce7db9fbee0dc27863c2207937d5c/Makefile#L9

Can you run and log the line to see what it shows